project/src/components/card/card.module.css

```diff
@@ -0,0 +1,3 @@
+.placeCard__type::first-letter{
+  text-transform: uppercase;;
```

Comment:
У нас есть тип апартов Private room, он будет корректно отображаться?
```suggestion
.placeCard__type::first-letter {
  text-transform: uppercase;
```

Reply:
![image](https://user-images.githubusercontent.com/92183078/180316459-0ede1565-9193-40cf-ac7f-9502ffb327dd.png)
да, отображается корректно